config/locales/models/personality.ja.yml

```
@@ -8,6 +8,8 @@ ja:
         nickname: ニックネーム
         image: 写真
         description: 自己紹介
+        affiliation_name: 所属(大)
```

```suggestion
        affiliation_name: 所属している学校/会社など
```
せめて説明は具体的な方がいいかなと思った！